app/src/main/res/layout-land/resume_lesson_fragment.xml

```diff
@@ -54,36 +55,58 @@
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@id/resume_lesson_chapter_title_text_view" />
 
-      <androidx.constraintlayout.widget.ConstraintLayout
+      <com.google.android.flexbox.FlexboxLayout
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:layout_marginStart="32dp"
+        android:layout_marginStart="28dp"
         android:layout_marginTop="32dp"
-        android:layout_marginEnd="32dp"
+        android:layout_marginEnd="28dp"
+        app:alignContent="flex_start"
+        app:alignItems="center"
+        app:flexDirection="row"
+        app:flexWrap="wrap"
+        app:justifyContent="space_evenly"
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@id/resume_lesson_chapter_description_text_view">
 
-        <Button
+        <com.google.android.material.button.MaterialButton
           android:id="@+id/resume_lesson_start_over_button"
-          style="@style/StartOverLessonButton"
-          android:layout_marginEnd="12dp"
+          android:layout_width="wrap_content"
+          android:layout_height="wrap_content"
+          android:background="@drawable/secondary_button_background"
+          android:gravity="center"
+          android:fontFamily="sans-serif-medium"
+          android:minWidth="144dp"
+          android:minHeight="@dimen/clickable_item_min_height"
+          android:textAllCaps="true"
+          android:textColor="@color/component_color_shared_secondary_button_background_trim_color"
+          android:textSize="14sp"
           android:text="@string/start_over_lesson_button"
-          app:layout_constraintBottom_toBottomOf="parent"
-          app:layout_constraintEnd_toStartOf="@+id/resume_lesson_continue_button"
-          app:layout_constraintHorizontal_chainStyle="spread"
-          app:layout_constraintStart_toStartOf="parent" />
+          app:backgroundTint="@null"
+          app:icon="@drawable/ic_start_over_24dp"
+          app:iconTint="@color/component_color_shared_secondary_button_background_trim_color"
+          app:iconGravity="textStart" />
+
 
-        <Button
+        <com.google.android.material.button.MaterialButton
```

Comment:
Could you explain why this change of button class was necessary? It might be worth it to add the explanation to the PR description.

Reply:
The use of material button has better control for the icon inside the button so that it can group icon and text together as responsive design guidelines suggested. Particularly the icon gravity part that attached the icon to the text.